README.md

```diff
@@ -131,11 +131,15 @@ You can find every supported distribution at the [official installation page](ht
 
 ## Development
 
+We encourage community contributions to Kong. To make sure it is a smooth
```

Comment:
Ok @Tieske sorry for that. Thank you for taking a look at my work. That will work fine for me.